test/cans/prod/cc0b5aa4-9f66-48d2-aa4f-ed019d1c9439/2019-05-15T222432.561000Z/manifest.json

```diff
@@ -0,0 +1,266 @@
+[
```

Comment:
Let's slim this can down so it is minimal and only contains what's needed for the test.